src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt

```diff
@@ -32,7 +39,8 @@ object LockMouseLook {
 
     @SubscribeEvent
     fun onChat(event: LorenzChatEvent) {
-        if (!event.message.startsWith("§aTeleported you to §r§aPlot")) return
+        println(!unlockMousePattern.matches(event.message))
```

Comment:
🔥

Reply:
i have no memory of even putting that there